<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImplantPlanning</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>911</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
     <widget class="QVTKOpenGLWidget" name="femurRenderer">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>291</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QVTKOpenGLWidget" name="tibiaRenderer">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>311</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenFemur">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Femur</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenTibia">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>350</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Tibia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveLandmarksFemur">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save landmarks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveLandmarksTibia">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save landmarks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenRealSurfaceFemur">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>430</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Real surface probe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenRealSurfaceTibia">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Real surface probe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenRealLandmarksFemur">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>460</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Real landmarks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenRealLandmarksTibia">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>460</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Real landmarks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalibrateFemur">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>520</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibrate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalibrateTibia">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>520</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibrate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Motion</string>
     </attribute>
     <widget class="QVTKOpenGLWidget" name="vtkRenderModel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>591</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="hSliderTIME">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>440</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenMotion">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load motion data</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnShowROI">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>410</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Show ROI only</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcDistances">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>710</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate distances</string>
      </property>
     </widget>
     <widget class="QVTKOpenGLWidget" name="vtkRenderChart">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>490</y>
        <width>581</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QVTKOpenGLWidget" name="vtkRenderTibia">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">/home/jeroen/Desktop/ProgramFiles/VTK-8.2.0/GUISupport/Qt/QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
